src/main/fc/rc.c

```diff
@@ -278,8 +278,7 @@ static void checkForThrottleErrorResetState(uint16_t rxRefreshRate)
     static int index;
     static int16_t rcCommandThrottlePrevious[THROTTLE_BUFFER_MAX];
 
-    const int rxRefreshRateMs = rxRefreshRate / 1000;
-    const int indexMax = constrain(THROTTLE_DELTA_MS / rxRefreshRateMs, 1, THROTTLE_BUFFER_MAX);
+    const int indexMax = constrain(THROTTLE_DELTA_MS*1000 / rxRefreshRate, 1, THROTTLE_BUFFER_MAX);
```

Comment:
In the SITL case, with UDP RC input, rxRefreshRate may be lower than 1000, which causes rxRefreshRateMs to be zero. 
This will then induce a segmentation fault. In original line 282.